src/components/Stats/StatsView.js

```diff
@@ -1,15 +1,13 @@
-import React from 'react';
-import { StatsBar } from 'bento-components';
-import { statsStyling, globalStatsData } from '../../bento/globalStatsData';
-
-const StatsView = ({ data }) => (
-  <>
-    <StatsBar
-      data={data}
-      globalStatsData={globalStatsData}
-      statsStyling={statsStyling}
-    />
-  </>
-);
-
-export default StatsView;
+import React from 'react';

```

Comment:
Are we using a Single Stats bar component on all the pages? Or Are we having variations of the Stats bar?

Reply:
@JoonLeeNIH

Reply:
Just one `StatsBar` component. `bento-frontend` has the `StatsView` wrapper to pass props to `StatsBar`, and various files use `StatsView`.